src/4.3_to_5.0/__test_resources__/legacyKpi.ts

```diff
@@ -31,6 +31,9 @@ export const legacyKpi: LegacyWidgetState = {
       configuration: {
         featuredValues: {
           locations: {
+            "": {
+              title: "Empty measure title",
+            },
```

Comment:
For this fix, I went with dropping titles that have an empty tuple key and ensuring the rest of the titles are still migrated. However, I'm not 100% sure how one can even land in this state. It doesn't seem like it's valid so I dropped it, but if it is better to ensure this kind of custom title is supported, I can adapt the change to ensure it's also migrated.